pkg/vulnsrc/vulnerability/vulnerability.go

```diff
@@ -22,11 +22,15 @@ var (
 		RubySec, RustSec, PhpSecurityAdvisories, NodejsSecurityWg, PythonSafetyDB}
 )
 
+var (
+	getDetailFunc = getDetail
+)
+
 func FillAndFilter(vulns []DetectedVulnerability, severities []Severity, ignoreUnfixed bool) []DetectedVulnerability {
 	ignoredIDs := getIgnoredIDs()
```

Comment:
I'm not sure why this isn't coming out in the diff but it looks like `getIgnoredIDs()` no longer takes in an argument for `ignoreFile`. Is that a flag we're going to deprecate? Would that break backwards compatibility with existing usage of the flag?

Reply:
<img width="1072" alt="image" src="https://user-images.githubusercontent.com/1254783/67047455-317e8e80-f0e7-11e9-9f9b-0fb817b40c20.png">

Reply:
I fixed it.